### ISSUE TITLE
.bashrc syntax error

### ISSUE THREAD

**Issue by mtancoigne** [DevBlend/zenias#13]
When logging in the VM, bash throws this error :

```
Welcome to Ubuntu 14.04.3 LTS (GNU/Linux 3.16.0-55-generic x86_64)

 * Documentation:  https://help.ubuntu.com/
Development Environment
Last login: Thu May 12 03:14:58 2016 from 10.0.2.2
-bash: $'\r': command not found
-bash: /home/vagrant/.bashrc: line 6: syntax error near unexpected token `$'in\r''
'bash: /home/vagrant/.bashrc: line 6: `case $- in
vagrant@trusty64:~$
```

wich correspond at this case in the .bashrc file :

``` bash
# If not running interactively, don't do anything
case $- in
    *i*) ;;
      *) return;;
esac
```

**Comment by alayek** [comment]
Thanks @mtancoigne for informing us. I will push a fix soon.

**Comment by koustuvsinha** [comment]
No issues for me (Ubuntu + Virtualbox). @alayek @byteknacker can you test this?

**Comment by byteknacker** [comment]
@mtancoigne thanks for creating this issue and letting us know about this bug. Would you like to join our slack channel where we actively exchange development details about this project?

**Comment by byteknacker** [comment]
@koustuvsinha testing now. I am doing it with zeus.py. But will also try another one just with vagrant up directly from the git clone of this repo.

**Comment by mtancoigne** [comment]
Guys, the issue appeared with a windows 10 host with vagrant 1.8.1 and virtualBox 5.0.18

Then I tested on a kubuntu 16.04 with vagrant 1.8.1 and virtualBox 5.0.18.

I had to install `virtualbox-ext-pack` to get USB2 support, but it went fine, no errors.

**Comment by alayek** [comment]
@mtancoigne If you can, please uninstall Virtualbox from Windows. Then install version 5.0.20 and latest extension pack. Then latest vagrant and try again.

**Comment by mtancoigne** [comment]
The issue remains on windows with vagrant 1.8.1 and Virtualbox 5.0.20

**Comment by alayek** [comment]
I see. Let me see what I can do.

**Comment by byteknacker** [comment]
@alayek  and @mtancoigne I got a similar issue at the very beginning of the project when vagrant up failed for minimal/trusty64. The reason was old version of virtualbox and most importantly the lack of the Extension Pack. @mtancoigne did you download the extension pack? I think we should add this to the dependencies list on README.md.

**Comment by alayek** [comment]
Without extension pack, the error would be different. But, yes, it might be older version of extension pack.

@mtancoigne please give another try with
- latest Virtualbox
- latest extension pack.
- latest vagrant

**Comment by byteknacker** [comment]
@alayek but how come it is a syntax error? I don't understand this shouldn't be happening. The box is the same for everyone. @mtancoigne can you get inside the vagrant ssh session and do some stuff normally there with terminal?

**Comment by byteknacker** [comment]
@mtancoigne please sign up here if you like to join our slack channel to collaborate in real time: https://johnwu.typeform.com/to/ifhehu

**Comment by alayek** [comment]
Closed by #14
